pages/container/media.page.tsx

```diff
@@ -0,0 +1,161 @@
+import ScreenshotArea from '../utils/screenshot-area';
+import SpaceBetween from '~components/space-between';
+import image169 from './images/16-9.png';
+import image43 from './images/4-3.png';
+import image916 from './images/9-16.png';
+
+import styles from './media.scss';
+
+type DemoContext = React.Context<
+  AppContextType<{
+    width: string;
+    height: string;
+    position: 'top' | 'side';
+    content: '16-9' | '4-3' | '9-16';
+  }>
+>;
+
+function ContainerPlayground(props: ContainerProps) {
+  const { urlParams } = useContext(AppContext as DemoContext);
+
+  return (
+    <Container
+      {...props}
+      media={{
+        content: (
+          <img
+            src={urlParams.content === '4-3' ? image43 : urlParams.content === '9-16' ? image916 : image169}
+            alt="A fun, accessible image"
+          />
+        ),
+        width: urlParams.width,
+        height: urlParams.height,
+        position: urlParams.position,
+      }}
+    />
+  );
+}
+
+function SettingsForm() {
+  const { urlParams, setUrlParams } = useContext(AppContext as DemoContext);
+  return (
+    <SpaceBetween direction="horizontal" size="m">
+      <FormField description="Only valid for 'side' position." label="Media width">
+        <Input
+          placeholder={'example: 30%'}
+          value={urlParams.width}
+          onChange={event => setUrlParams({ width: event.detail.value })}
+        />
+      </FormField>
+      <FormField description="Only valid for 'top' position." label="Media height">
+        <Input
+          placeholder={'example: 200px'}
+          value={urlParams.height}
+          onChange={event => setUrlParams({ height: event.detail.value })}
+        />
+      </FormField>
+      <FormField label="Media position">
+        <RadioGroup
+          items={[
+            {
+              value: 'top',
+              label: 'Top',
+            },
+            {
+              value: 'side',
+              label: 'Side',
+            },
+          ]}
+          onChange={({ detail }) => setUrlParams({ position: detail.value as 'top' | 'side' })}
+          value={urlParams.position}
+        />
+      </FormField>
+      <FormField label="Content">
+        <RadioGroup
+          items={[
+            {
+              value: '16-9',
+              label: '16:9 (landscape) image',
+            },
+            {
+              value: '4-3',
+              label: '4:3 (squareish) image',
+            },
+            {
+              value: '9-16',
+              label: '9:16 (portrait) image',
+            },
+          ]}
+          onChange={({ detail }) => setUrlParams({ content: detail.value as '16-9' | '4-3' | '9-16' })}
+          value={urlParams.content}
+        />
+      </FormField>
+    </SpaceBetween>
+  );
+}
+
+export default function MediaContainers() {
+  return (
+    <article>
+      <h1>Media containers</h1>
+
+      <ScreenshotArea>
+        <SpaceBetween size="l">
+          <SettingsForm />
```

Comment:
Pro tip. You can wrap screenshot area only around the actual content you want to take a screenshot of. No need to include settings form.

Reply:
Settings are also reflected in URL params and this info is available in the report.

Taking screenshot of a setting form may cause problems. If we change any color in the checkbox component, it will produce an extra bunch of "failed" screenshots, more manual work for the team to review.

There is a guideline for the tests to be atomic and precise against the subject being tested. Screenshot tests are not an exception

Reply:
Fix: https://github.com/cloudscape-design/components/pull/1216/